pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -226,6 +226,8 @@ var (
 			// local-with-fallback annotation for kube-proxy (see
 			// <https://bugzilla.redhat.com/show_bug.cgi?id=1960284>).
 			localWithFallbackAnnotation,
+			// Sets annotation for NLB in AWS
+			AWSLBTypeAnnotation,
```

Comment:
This is needed for #790 so that the operator correctly _updates_ the annotation, but why is it needed for _this_ PR?  This PR changes the behavior when the operator _creates_ an ingresscontroller, in which case the operator should also be _creating_ a new service rather than updating an existing one.

Reply:
yeah i will remove it.